ReCaptchaUi/Model/ValidationConfigResolver.php

```diff
@@ -33,22 +36,18 @@ class ValidationConfigResolver implements ValidationConfigResolverInterface
     /**
      * @param CaptchaTypeResolverInterface $captchaTypeResolver
      * @param ValidationConfigProviderInterface[] $validationConfigProviders
-     * @throws InputException
      */
     public function __construct(
         CaptchaTypeResolverInterface $captchaTypeResolver,
         array $validationConfigProviders = []
     ) {
         $this->captchaTypeResolver = $captchaTypeResolver;
-
-        foreach ($validationConfigProviders as $validationConfigProvider) {
-            if (!$validationConfigProvider instanceof ValidationConfigProviderInterface) {
-                throw new InputException(
-                    __('Validation config provider must implement %1.', [ConfigProviderInterface::class])
-                );
-            }
-        }
-        $this->validationConfigProviders = $validationConfigProviders;
+        $this->validationConfigProviders = array_combine(
```

Comment:
Same with this one, let's switch it back

Reply:
we could get rid of this totally, as the get method is signed, we are sure that the correct type should be returned. Developers must follow the contrat of the doc anyway

Reply:
I don't think we even need to have such validation. There is an implicit contract. I agree we shouldn't add noise when it has no value.

Reply:
@nathanjosiah